public/features.json

```diff
@@ -15,6 +15,10 @@
     "id": "tank",
     "title": "Main tank"
   },
+  {
+    "id": "offtank",
```

Comment:
I don't really like this conception since any tank or maybe even a bruiser can be offtank.
But if you think that it should be present I'm ok with it. We can discuss this in future with more people included.

Reply:
Yes, but not every offtank can be a main tank (e.g. Zarya can't MT). I agree that mostly this is very illusive border, but it's not playing role in counter-picks, at the same time it allows you to specify if you're already have an MT in the draft

Reply:
This is exactly my point. Zarya is not an off-tank. She is just bruiser/support. So is Artanis. And Sonya.

These are just frontline heroes. If you pick E.T.C and Anub, Anub will be an offtank.

Reply:
Hmm.. That's not exactly what i meant. But ok, to avoid confusion i'll remove it